changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,54 @@ Ansible Netcommon Collection Release Notes
 .. contents:: Topics
 
 
+v1.1.2
+======
+
+Release Summary
+---------------
+
+Rereleased 1.1.1 with updated changelog.
+
+v1.1.1
+======
+
+Release Summary
+---------------
+
+Rereleased 1.1.0 with regenerated documentation.
+
+v1.1.0
+======
+
+Major Changes
+-------------
+
+- Add libssh connection plugin and refactor network_cli (https://github.com/ansible-collections/ansible.netcommon/pull/30)
+
+Minor Changes
+-------------
+
+- Add changelogs for 1.0.0 release (https://github.com/ansible-collections/ansible.netcommon/pull/67)
```

Comment:
This seems to be redundant and made its way in because #67 had a changelog for itself?